src/components/register/index.css

```diff
@@ -0,0 +1,77 @@
+.loginContainer {
```

Comment:
so this is register/index but its called logincontainer. and i'm guessing you just copy-pasted the code. css should be reusable, so if two things are using the same styles, can you find a way to reuse the classes?

Reply:
sure, I can lift the CSS up to RegisterLogin parent components stylesheet